docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -0,0 +1,423 @@
+# Prerequisite
+
+## Possible fields
+- `parent_id`
+- `name`
+- `opening_status`
+- `org_unit_type_id`
+- `groups`
+- `geo_point`
+- `shape`
+- `instances`
+
+# POST /api/orgunits/changes/?app_id=..
```

Comment:
I'm thinking of naming the model `OrgUnitUpsertRequest`. Sounds better suited for both INSERT and UPDATE operations.

WDYT of `/api/orgunits/upsert/?app_id=..`

Reply:
I don't have any opinion on this. To my understanding, `upsert` is a made-up word by PostgreSQL, but it's clear. @madewulf, what is your take on this?

Reply:
Clarified in meeting. `ChangeRequest` is good enough.